src/test/java/gov/healthit/chpl/aqa/stepDefinitions/ChplDownloadSteps.java

```diff
@@ -63,6 +43,10 @@ public ChplDownloadSteps() {
         if (StringUtils.isEmpty(url)) {
             url = "http://localhost:3000/";
         }
+        if (StringUtils.isEmpty(downloadPath)) {
+           downloadPath = "\\target";
```

Comment:
My "target" directory has stuff, and likely will on any system that's running this test. I like using target as a starting point, but would suggest something like `\\target\\download-files` instead